tests/functional/disaster-recovery/regional-dr/test_deploy_rdr.py

```
@@ -0,0 +1,523 @@
+
+from ocs_ci.deployment.deployment import Deployment
+from ocs_ci.framework import config
+from ocs_ci.helpers.sanity_helpers import Sanity, SanityExternalCluster
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.utils import get_non_acm_cluster_config
+from ocs_ci.utility.utils import ceph_health_check
+
+log = logging.getLogger(__name__)
+
+
+def test_deploy_rdr():
```

Ruff (0.15.12)

[warning] Too many branches (47 > 12)

(PLR0912)